### PR TITLE
Add 'path_prompt_prefix' option

### PR DESCRIPTION
**Pull request by CKolkey**
If enabled, will update the prompt with the relative path from the CWD. Defaults to `false`
Screenshot 2023-03-22 at 15 15 01
Screenshot 2023-03-22 at 15 15 14

### CODE REVIEW

File: doc/telescope-file-browser.txt [hunk 1]
```diff
@@ -158,6 +158,9 @@ fb_picker.file_browser({opts})  *telescope-file-browser.picker.file_browser()*
         {git_status}        (boolean)         show the git status of files
                                               (default: true if `git`
                                               executable can be found)
+        {prompt_path}       (boolean)         Show the current relative path
```
Comment (fdschmidt93) [inline doc/telescope-file-browser.txt:161]: Docs are generated from lua annotation. This needs to go to `picker.lua`.

**Review by fdschmidt93 — APPROVED** [review]
LGTM thanks :) briefly tried it and only needs annotations in lua to correctly generate the docs. Probably best to revert the doc change and then annotate in lua so CI correctly does the job.

**Comment by CKolkey** [comment]
Alright, let's see if that does it :)

**Comment by fdschmidt93** [comment]
Thanks :) I'll have to trigger docgen manually on master after merging. It's often not running especially with first time contributors.

**Comment by CKolkey** [comment]
Ahh, got it. Cool, thanks!

**Pull request merged**